os/macosx/Info.plist.in

```diff
@@ -33,5 +33,7 @@
         <string>True</string>
         <key>LSMinimumSystemVersion</key>
         <string>10.13.0</string>
+        <key>LSApplicationCategoryType</key>
+        <string>public.app-category.games</string>
```

Comment:
Isn't `public.app-category.simulation-games` a more specific type for OpenTTD? Or does that not enable game mode?

Reply:
All game categories enable the mode, they just get more specific. I didn't really find one that fits well at first, but this would work too.